aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -553,6 +743,35 @@ module aptos_framework::account {
             offerer_account_resource,
             new_auth_key
         );
+        if (new_scheme == MULTI_ED25519_SCHEME) {
+            let len = vector::length(&cap_update_table);
+            let signature_bitmap = vector::slice(&cap_update_table, len - 4, len);
+            event::emit(KeyRotationToMultiPublicKey {
+                account: rotation_cap_offerer_address,
+                verified_public_key_bit_map: signature_bitmap,
+                multi_public_key_scheme: new_scheme,
+                multi_public_key: new_public_key_bytes,
+                authentication_key: new_auth_key,
+            });
+        }
```

Comment:
I don't think this belongs in this PR.

Let's not pollute other changes into a change specifically for Keyless Multikey backups